src/RootWithIntl.js

```diff
@@ -21,6 +22,10 @@ import OverlayContainer from './components/OverlayContainer';
 import getModuleRoutes from './moduleRoutes';
 import { stripesShape } from './Stripes';
 
+const wrapInErrorBoundary = (component) => (
```

Comment:
I'm defining `wrapInErrorBoundary` here purely for readability sake. There's so many brackets floating around that it made the code cleaner IMO.